app/scripts/metamask-controller.js

```diff
@@ -3438,6 +3438,18 @@ export default class MetamaskController extends EventEmitter {
     });
   }
 
+  handleWatchAssetRequest = (asset, type, origin) => {
+    switch (type) {
+      case 'ERC20':
+        return this.tokensController.watchAsset(asset, type);
+      case 'ERC721':
+      case 'ERC1155':
```

Comment:
should the constants from `@metamask/controller-utils` be used here?

Reply:
yes we can do this!

Reply:
done [here](https://github.com/MetaMask/metamask-extension/pull/19454/commits/6a893762979dddc49349da0491d04e4570651e69)